resources/schema.sql

```diff
@@ -14,12 +14,15 @@ CREATE TABLE `endpoints` (
 );
 
 CREATE TABLE `selectors` (
-	`id` VARCHAR(255) NOT NULL,
-	`endpoint_id` VARCHAR(255) NOT NULL,
-	`selector` VARCHAR(150) NOT NULL,
-	`alias` VARCHAR(150) NOT NULL,
-	PRIMARY KEY (`id`)
-);
+  `id` varchar(256) NOT NULL,
```

Comment:
Is there a reason you put the varchars back to 256 characters? The extra bit makes it use another byte in total